app/book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @rentals = []
+    @title = title
+    @author = author
+  end
+end
```

Comment:
Kindly include a method for adding rentals. Check out example [here](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship)